AUDIO/BlockAllSpeechFromPed.md

```diff
@@ -1,11 +1,11 @@
 ---
 ns: AUDIO
 ---
-## _0xF8AD2EED7C47E8FE
+## BLOCK_​ALL_​SPEECH_​FROM_​PED
```

Comment:
From [CONTRIBUTING.md](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) under naming convention:

Community-named natives must be prefixed by an underscore, e.g. _EXAMPLE_NATIVE

- Community-named natives have names that do not match the hash of the native.